### ISSUE TITLE
Error on running $ nodemon ./.server/server.js. 

### ISSUE THREAD

**Issue by hadirsa** [fulls1z3/universal#317]
```
[*] Regression (a behavior that used to work and stopped working in a new release)
```
**I get error when i build and run `nodemon ./.server/server.js`. the web page is white and it stuck on loading.**
```
$ nodemon ./.server/server.js
[nodemon] 1.14.11
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `node ./.server/server.js`
Express server listening on http://localhost:8000
Unhandled Promise rejection: mime.lookup is not a function ; Zone: <root> ; Task: Promise.then ; Value: TypeError: mime.lookup is not a function
    at ServerResponse.contentType (C:\Users\hadirasouli\sample\universal-master\.server\server.js:215399:12)
    at ServerResponse.send (C:\Users\hadirasouli\sample\universal-master\.server\server.js:214952:14)
    at done (C:\Users\hadirasouli\sample\universal-master\.server\server.js:215811:10)
    at C:\Users\hadirasouli\sample\universal-master\.server\server.js:216939:17
    at ZoneDelegate.invoke (C:\Users\hadirasouli\sample\universal-master\.server\server.js:163115:26)
    at Zone.run (C:\Users\hadirasouli\sample\universal-master\.server\server.js:162865:43)
    at C:\Users\hadirasouli\sample\universal-master\.server\server.js:163585:57
    at ZoneDelegate.invokeTask (C:\Users\hadirasouli\sample\universal-master\.server\server.js:163148:31)
    at Zone.runTask (C:\Users\hadirasouli\sample\universal-master\.server\server.js:162915:47)
    at drainMicroTaskQueue (C:\Users\hadirasouli\sample\universal-master\.server\server.js:163322:35) TypeError: mime.lookup is not a function
    at ServerResponse.contentType (C:\Users\hadirasouli\sample\universal-master\.server\server.js:215399:12)
    at ServerResponse.send (C:\Users\hadirasouli\sample\universal-master\.server\server.js:214952:14)
    at done (C:\Users\hadirasouli\sample\universal-master\.server\server.js:215811:10)
    at C:\Users\hadirasouli\sample\universal-master\.server\server.js:216939:17
    at ZoneDelegate.invoke (C:\Users\hadirasouli\sample\universal-master\.server\server.js:163115:26)
    at Zone.run (C:\Users\hadirasouli\sample\universal-master\.server\server.js:162865:43)
    at C:\Users\hadirasouli\sample\universal-master\.server\server.js:163585:57
    at ZoneDelegate.invokeTask (C:\Users\hadirasouli\sample\universal-master\.server\server.js:163148:31)
    at Zone.runTask (C:\Users\hadirasouli\sample\universal-master\.server\server.js:162915:47)
    at drainMicroTaskQueue (C:\Users\hadirasouli\sample\universal-master\.server\server.js:163322:35)
```

**Is there any command that i should run before starting server?**

**Comment by hadirsa** [comment]
I wonder if it works for the others!!

**Comment by vlados** [comment]
@hadirsa This function is not part of the package. Search which package is doing this and try to debug it

**Comment by fulls1z3** [comment]
@vlados thanks for catching the issue 👍 
@hadirsa please try to share your package.json with us, maybe we can notice the package.

happy valentines everyone ❤️

**Comment by kushwahashiv** [comment]
this do not works for me as well. not sure if there is a way to generate it?

**Comment by vlados** [comment]
@kushwahashiv try
`npm install mime@1.2.11 --save`
